### PR TITLE
#690 Upgrading service instances,but gateway choose old instance for request

### PR DESCRIPTION
**Pull request by chengyouling**
Signed-off-by:YouLing Cheng <spadgerlin888@163.com>

### CODE REVIEW

File: spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java [hunk 1]
```diff
@@ -35,7 +36,14 @@ public class ServiceCombServerListUpdater implements ServerListUpdater {
...
   private UpdateAction updateAction;
 
+  private PollingServerListUpdater pollingServerListUpdater;
+
+  private static long INITIAL_DELAY_TIME = 1000;
+
+  private static long REFRESH_INITIAL_TIME = 5 * 1000;
+
   public ServiceCombServerListUpdater() {
+    pollingServerListUpdater = new PollingServerListUpdater(INITIAL_DELAY_TIME, REFRESH_INITIAL_TIME);
```
Comment (Shoothzj) [inline spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java:46]: It's better to make `PollingServerListUpdater` final

Comment (chengyouling) [reply spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java:46]: pollingServerListUpdater isNot paramer，just init when construction method execute，is still has any problems？

Comment (Shoothzj) [reply spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java:46]: line39, this field wii never change, So it's better to make this field final.

Comment (chengyouling) [reply spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java:46]: ok,thank you!

**Comment by liubao68** [comment]
see https://github.com/huaweicloud/spring-cloud-huawei/pull/692

这个PR解决不了服务被删除后打印大量日志的问题。

**Pull request closed without merging**